src/js/components/react-file-base64.js

```diff
@@ -10,12 +10,12 @@ import ReactDOM from 'react-dom';
 
 class FileBase64 extends React.Component {
 
-  constructor() {
-    super()
+  constructor(props) {
+    super(props)
     this.state = {
       files: []
     }
-    this.props = {
+    props = {
```

Comment:
This will make it so it doesn't affect the component props but just the variable props. The right way to do this is as I did in the PR #4 - Using `defaultProps`.